buildSrc/subprojects/profiling/src/main/kotlin/org/gradle/gradlebuild/profiling/buildscan/BuildScanPlugin.kt

```diff
@@ -241,32 +208,6 @@ open class BuildScanPlugin : Plugin<Project> {
         }
     }
 
-    private
-    fun Project.extractVcsData() {
-        buildScan {
-
-            if (!isCiServer) {
-                background {
-                    setCommitId(execAndGetStdout("git", "rev-parse", "--verify", "HEAD"))
-                }
-            }
-
-            background {
-                execAndGetStdout("git", "status", "--porcelain").takeIf { it.isNotEmpty() }?.let { status ->
-                    tag("dirty")
-                    value("Git Status", status)
-                }
-            }
-
-            background {
-                execAndGetStdout("git", "rev-parse", "--abbrev-ref", "HEAD").takeIf { it.isNotEmpty() && it != "HEAD" }?.let { branchName ->
-                    tag(branchName)
-                    value("Git Branch Name", branchName)
-                }
-            }
-        }
-    }
-
     private
     fun Project.extractBuildCacheData() {
```

Comment:
Can we move that to the plugin as well? `extractWatchFsData` would also be great for the plugin, though this can be done later as well.

Reply:
Done with `0.2` plugin version.